exchanges/bybit/bybit_test.go

```diff
@@ -85,3521 +80,3401 @@ func TestGetKlines(t *testing.T) {
-	if err != nil {
-		t.Error(err)
-	}
-
-	pair1, err := currency.NewPairFromString("BTCUSD")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	var getOrdersRequestCMF = order.MultiOrderRequest{
-		Pairs:     currency.Pairs{pair1},
-		AssetType: asset.CoinMarginedFutures,
-		Type:      order.AnyType,
-		Side:      order.AnySide,
-	}
-
-	_, err = b.GetOrderHistory(context.Background(), &getOrdersRequestCMF)
-	if err != nil {
-		t.Error(err)
-	}
-
-	pair2, err := currency.NewPairFromString("BTCUSDZ22")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	var getOrdersRequestFutures = order.MultiOrderRequest{
-		Pairs:     currency.Pairs{pair2},
-		AssetType: asset.Futures,
-		Type:      order.AnyType,
-		Side:      order.AnySide,
-	}
-
-	_, err = b.GetOrderHistory(context.Background(), &getOrdersRequestFutures)
-	if err != nil {
-		t.Error(err)
-	}
-
-	pair3, err := currency.NewPairFromString("BTCPERP")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	var getOrdersRequestUSDC = order.MultiOrderRequest{
-		Pairs:     currency.Pairs{pair3},
-		AssetType: asset.USDCMarginedFutures,
-		Type:      order.AnyType,
-		Side:      order.AnySide,
+	if mockTests {
+		t.Skip(skipAuthenticatedFunctionsForMockTesting)
 	}
-
-	_, err = b.GetOrderHistory(context.Background(), &getOrdersRequestUSDC)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
+	_, err := b.ModifySubAPIKey(context.Background(), &SubUIDAPIKeyUpdateParam{})
+	if !errors.Is(err, errNilArgument) {
+		t.Fatalf("expected %v, got %v", errNilArgument, err)
+	}
+	_, err = b.ModifySubAPIKey(context.Background(), &SubUIDAPIKeyUpdateParam{
+		ReadOnly: 0,
+		IPs:      []string{"*"},
+		Permissions: map[string][]string{
+			"ContractTrade": {},
+			"Spot":          {"SpotTrade"},
+			"Wallet":        {"AccountTransfer"},
+		},
+	})
 	if err != nil {
 		t.Error(err)
 	}
 }
 
-func TestGetWithdrawalsHistory(t *testing.T) {
+func TestDeleteSubUID(t *testing.T) {
 	t.Parallel()
+	if mockTests {
+		t.Skip(skipAuthenticatedFunctionsForMockTesting)
+	}
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.CoinMarginedFutures)
+	err := b.DeleteSubUID(context.Background(), "")
+	if !errors.Is(err, errMemberIDRequired) {
+		t.Errorf("expected %v, got %v", errMemberIDRequired, err)
+	}
+	err = b.DeleteSubUID(context.Background(), "1234")
 	if err != nil {
 		t.Error(err)
 	}
-
-	_, err = b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
-	if err == nil {
-		t.Error("GetWithdrawalsHistory() Spot Expected error")
-	}
 }
 
-func TestGetServerTime(t *testing.T) {
+func TestDeleteMasterAPIKey(t *testing.T) {
 	t.Parallel()
-
-	_, err := b.GetServerTime(context.Background(), asset.CoinMarginedFutures)
+	if mockTests {
+		t.Skip(skipAuthenticatedFunctionsForMockTesting)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
+	err := b.DeleteMasterAPIKey(context.Background())
```

Comment:
This test tripped me up with running all tests 😆

Reply:
Should we add another filter in case someone dares to enable **_canManipulateRealOrders_**

Reply:
Nah, its on me setting `canManipulateRealOrders` to `true` and then dealing with these consequences. I was just surprised to see and then had to create a new one -.-